public/index.php

```diff
@@ -16,6 +16,14 @@
 	));
 });
 
+$klein->respond('/create', function() use ($twig, $crud){
+	$crud->create($_POST['name'], $_POST['surname'], $_POST['indexno'], $_POST['address']);
+	$getAll = $crud->getAll();
```

Comment:
we do not need this in our 'create' route nor do we need to show the end result, we can call this method after we have created our student